### DISCUSSION TITLE
unresolved external symbol emscripten_GetProcAddress in wasm example build on windows 11

### DISCUSSION THREAD

**Discussion by LiamPClancy** [rust-skia/rust-skia#803] [Q&A]
I"ve attempted this build with emscripten versions 3.1.1 and 3.1.10 and the issue seems to exist in both. 

While attempting to build the wasm example 
using either
`cargo build -vv`
and
`cargo run -vv`

i'm getting a error
`  = note: wasm_example.6mt6hskmh9dqqma.rcgu.o : error LNK2019: unresolved external symbol emscripten_GetProcAddress referenced in function _ZN12wasm_example7init_gl28_$u7b$$u7b$closure$u7d$$u7d$17h4d9f723d936b948fE                          
          C:\dev\rust-skia\wasm-example\target\debug\deps\wasm_example.exe : fatal error LNK1120: 1 unresolved externals
                                                                                                                        `
I'd like to add that I don't have basically no experience in rust, C, or C++ so it might be that I'm missing something really basic. But wanted to report it incase it is an issue, and also if not, hopefully a tip on how to resolve the issue. 

I've attached the a copy of what I think are the relevant parts of the the build log.
let me know if you require further information.                                                                                                                     

[buildLog.txt](https://github.com/rust-skia/rust-skia/files/8754821/buildLog.txt)

**Comment by enricobuehler** [comment]
Make sure you're running `cargo build --target wasm32-unknown-emscripten` otherwise the build wont work. You also got to set the env variables as the readme says, you can do this by creating a .cargo folder at the project root and creating a config.toml inside with the following content:

```
[env]
EMSDK = "path/to/emscriptensdk"
EMCC_CFLAGS = "-s ERROR_ON_UNDEFINED_SYMBOLS=0 -s MAX_WEBGL_VERSION=2"

```
